packages/expo-updates/src/Updates.ts

```diff
@@ -318,3 +357,66 @@ export function setUpdateURLAndRequestHeadersOverride(
 ): void {
   ExpoUpdates.setUpdateURLAndRequestHeadersOverride(configOverride);
 }
+
+/**
+ * Sets the configuration for the reload screen that will be displayed during `reloadAsync()` calls.
+ * By default, the reload screen shows a centered loading spinner on a white background.
+ *
+ * @param options Configuration options for customizing the reload screen appearance.
+ *
+ * @example
+ * ```ts
+ * import * as Updates from 'expo-updates';
+ *
+ * // Set a custom background color and spinner color
+ * Updates.setReloadScreenOptions({
+ *   backgroundColor: '#1a1a1a',
+ *   spinner: {
+ *     color: '#ffffff'
+ *   }
+ * });
+ *
+ * // Use a custom image instead of spinner
+ * Updates.setReloadScreenOptions({
+ *   backgroundColor: '#ffffff',
+ *   image: require('./assets/loading.png'),
+ *   imageResizeMode: 'contain'
+ * });
+ * ```
+ */
+export function setReloadScreenOptions(options: ReloadScreenOptions): void {
+  const resolvedOptions = resolveReloadScreenOptions(options);
+  ExpoUpdates.setReloadScreenOptions(resolvedOptions);
+}
```

Comment:
nit: i guess we don't keep persistency for this reload screen options. would be good to document that people should always call this function before reloadAsync. otherwise, after reload, we will fallback to the default white background screenshot

Reply:
sorry i did not express it clearly. for the persistency i mean we don't keep the reload screen options in file system. if people don't call `setReloadScreenOptions` in next launch, it will fallback to default white background screen.

then we will document that people should call `setReloadScreenOptions` in each time before calling reloadAsync

Reply:
Ah, I see. Now I've added the options to `reloadAsync` so I would expect users will call that in the same fashion each time. For example
```typescript
Updates.reloadAsync({
  backgroundColor: "#000000",
  spinner: {
    color: "#ffffff"
  }
})
```
But we can call this out in the docs also

Reply:
Could we add these options to `showReloadScreen` to avoid needing `setReloadScreenOptions` altogether?